scripts/translation/qaxml.a.php

```diff
@@ -74,11 +74,24 @@
         print "\n";
 }
 
-function extractTriple( array $list )
+function extractTriple(array $list)
 {
     $ret = array();
-    foreach( $list as $elem )
-        foreach( $elem->attributes as $attrib )
-            $ret[] = "{$elem->nodeName} {$attrib->nodeName} {$attrib->nodeValue}";
+    try {
+        foreach ($list as $elem) {
+            if ($elem->hasAttributes()) {
```

Comment:
I would like to see some performance measurements here. See caveat in Conversation.